tests/testthat/test-utils-format.R

```diff
@@ -0,0 +1,47 @@
+context("utils-format")
+
+test_that("tibble_width returns user-input width,
+          then tibble.width option, then width option", {
+  test_width <- rpois(1, 100) # Randomized test value
```

Comment:
A randomized test value gives little gain here. If the test fails, we can't really reproduce it (because set.seed() is not called). If you do call set.seed(), you can also use a constant right away.